samples/bootloader/src/main.c

```diff
@@ -12,19 +12,30 @@
 #include <bl_storage.h>
 #include <bl_boot.h>
 #include <bl_validation.h>
+#include <nrfx_nvmc.h>
 
-#if defined(CONFIG_HW_UNIQUE_KEY)
+#if defined(CONFIG_HW_UNIQUE_KEY_LOAD)
 #include <init.h>
 #include <hw_unique_key.h>
 
+#define HUK_FLAG_OFFSET 0xFFC /* When this word is set, expect HUK to be written. */
+
 int load_huk(const struct device *unused)
 {
 	(void)unused;
 
 	if (!hw_unique_key_is_written(HUK_KEYSLOT_KDR)) {
+		uint32_t huk_flag = PM_HW_UNIQUE_KEY_PARTITION_ADDRESS + HUK_FLAG_OFFSET;
```

Comment:
Is this an address or a flag? Should we consider renaming the variable if its an address? `huk_flag_address` ?

Reply:
An address. changed to `huk_flag_addr`